components/automate-ui/src/app/components/notifications/notifications.component.html

```diff
@@ -1,14 +1,7 @@
 <div class="notifications">
   <ng-container *ngFor="let notification of notifications">
-    <chef-notification *ngIf="notification.type === 'error'"
-      [type]="notification.type === 'license' ? 'error' : notification.type"
-      [timeout]="notification.timeout" (dismissed)="handleNotificationDismissal(notification.id)">
-      {{ notification.message }}
-    </chef-notification>
-  </ng-container>
-  <ng-container *ngFor="let notification of notifications">
-    <chef-notification *ngIf="notification.type === 'info'"
-      [type]="notification.type === 'license' ? 'error' : notification.type"
+    <chef-notification *ngIf="notification.type !== 'license'"
+      [type]="notification.type"
       [timeout]="notification.timeout" (dismissed)="handleNotificationDismissal(notification.id)">
       {{ notification.message }}
     </chef-notification>
```

Comment:
Not changing anything here; logic should be exactly equivalent. Here's why:
In the original code there's an `ngIf` looking for `error` and another looking for `info`. So neither of them will render for type `license`, so it is sufficient to check for `!== license`, delete the check for `type === license` (which would never have been true), and use a single block because the two original blocks are identical.